### PR TITLE
PA-29 - dummy pets now in - 10 in total, 5 cats, 5 dogs

### PR DESCRIPTION
**Pull request by hester-g**

### CODE REVIEW

File: app/src/main/java/uk/ac/bath/petmatch/Database/DbHelper.java [hunk 1]
```diff
@@ -101,9 +101,19 @@ public void generateSampleData() {
...
         goodBoy = new Pet("Good boy", "He's the goodest boy", dummyBath, labrador);
+        roz = new Pet("Roz", "Is good at herding cows", bath, husky);
+        artem = new Pet("Artem", "Needs exercising often", dummyBath, beagle);
+        jan = new Pet("Jan", "Only eats pedigree food", bath, airedale);
+        josh = new Pet("Josh", "Likes it when you play soundtracks", dummyBath, chihuahua);
+        // dogs
         mcGonagall = new Pet("Minerva McGonagall", "Have a biscuit, Potter.", bath, tabby);
+        hagrid = new Pet("Hagrid", "Very hairy", dummyBath, ragdoll);
+        malfoy = new Pet("Draco Malfoy", "Weaselike in appearance", bath, siamese);
+        dumbledore = new Pet("Albus Percival Wulfric Brian Dumbledore", "Don't pity the dead, pity the living", dummyBath, britishshorthair);
```
Comment (novotnej) [inline app/src/main/java/uk/ac/bath/petmatch/Database/DbHelper.java:115]: :heart_eyes:

Comment (hester-g) [reply app/src/main/java/uk/ac/bath/petmatch/Database/DbHelper.java:115]: 🐱 🐶

Comment (novotnej) [reply app/src/main/java/uk/ac/bath/petmatch/Database/DbHelper.java:115]: @Jabzilla can you then please use this picture:

Dumblecat

**Review by novotnej — CHANGES_REQUESTED** [review]
I don't understand why there is no smelly cat but alright

Please increase the DATABASE_VERSION in DBHelper - generateSampleData is only run when version changes

**Review by novotnej — APPROVED** [review]

**Pull request merged**